example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Alexander Matthes, Benjamin Worpitz, Erik Zenker, Matthias Werner
+/* Copyright 2019-2021 Alexander Matthes, Benjamin Worpitz, Erik Zenker, Matthias Werner, Bernhard Manfred Gruber
```

Comment:
I think just having 2021 here is enough.

Reply:
I looked at other files and I have seen the date ranges used. I don't care how to write it, but it should be consistent.